qemu_build

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+BASENAME=bitnami/minideb
+pub_key_dir="$(mktemp -d)"
+
+do_sudo() {
+  if [[ "0" == "$(id --user)" ]]; then
+    "$@"
+  else
+    sudo "$@"
+  fi
+}
+
+do_ssh() {
+  ssh -o "StrictHostKeyChecking=no" -o "UserKnownHostsFile=/dev/null" root@localhost -t -p 5555 -i "$pub_key_dir/id_rsa" "$@"
+}
+
+finish() {
+  echo "Shutting down QEMU..."
+
+  n=0
+  until [ "$n" -ge 15 ]
+  do
+     do_ssh "true" && break
+     n=$((n+1))
+     sleep 30
+  done
+
+  do_ssh "poweroff" || true
+
+  sleep 5
+  n=0
+  until [ "$n" -ge 5 ]
+  do
+     kill -0 "$PID" && break
+     n=$((n+1))
+     sleep 5
+  done
+
+  kill -9 "$PID" || true
+  rm -f "$IMAGE_FILE" "$PIDFILE"
+}
+
+if [[ ! -f /etc/debian_version ]]; then
+  echo "minideb can currently only be built on debian based distros, aborting..."
+  exit 1
+fi
+
+if [ -z "$1" ]; then
+    echo "You must specify the dist to build"
+    exit 1
+fi
+
+DIST=$1
+PLATFORM=${2:-amd64}
+
+while do_sudo fuser /var/{lib/{dpkg,apt/lists},cache/apt/archives}/lock >/dev/null 2>&1; do
+    sleep 1
+done
```

Comment:
suggestion: This is done in the travis file, so also doing it here seems weird, suggest removing it unless there's a reason it needs to be repeated here.

Reply:
`qemu_build` file is not intended to be executed on travis.
I often encountered a lock issue while working on this executing `apt-get update`, so I copied this while loop from travis file. Is not really required, so can be removed.

Reply:
Ah, I figured this out later and that makes sense, thanks.